Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1068,6 +1069,10 @@ pair<string, MolDraw2D::OrientType> MolDraw2D::getAtomSymbolAndOrientation(
     symbol = "";
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
+  } else if (drawOptions().atomLabelDeuteriumTritium && atom.getAtomicNum() == 1 &&
```

Comment:
I'll eagerly await the pull request indicating that all relevant non-C++11 compilers have been eliminated. :-)